feat: Add support for Spicepod version v2 (#9583)
spiceai/spiceai: docs/release_notes/rc/v1.0.0-rc.4.md

    name: table
```

> - **Spicepod Spec Version:** Spicepod spec version `v1` is now by default. `v1beta1` is no longer supported.

Author: ```suggestion
- **Spicepod Spec Version:** Spicepod spec version `v1` is now by default. `v1beta1` will continue to work.
```
